config/swagger.yaml

```diff
@@ -584,6 +584,16 @@ paths:
           description: 'Hash of the block to fetch'
           required: true
           type : string 
+        - in: query
```

Comment:
If we want a complex object, we could probably put it in the body.
We could then also use enums to specify the different types.

Reply:
Answered [here](https://github.com/aeternity/epoch/pull/661#discussion_r165070869)